src/llama_stack_client/lib/agents/agent.py

```diff
@@ -35,7 +34,32 @@ def create_session(self, session_name: str) -> int:
         self.sessions.append(self.session_id)
         return self.session_id
 
-    async def create_turn(
+    def _has_tool_call(self, chunk):
+        if chunk.event.payload.event_type != "turn_complete":
+            return False
+        message = chunk.event.payload.turn.output_message
+        if len(message.tool_calls) == 0:
+            return False
+        if message.stop_reason == "out_of_tokens":
+            return False
+        return True
+
+    async def _async_run_tool(self, chunk):
+        message = chunk.event.payload.turn.output_message
+        tool_call = message.tool_calls[0]
+        if tool_call.tool_name not in self.custom_tools:
+            return ToolResponseMessage(
+                call_id=tool_call.call_id,
+                tool_name=tool_call.tool_name,
+                content=f"Unknown tool `{tool_call.tool_name}` was called. Try again with something else",
```

Comment:
The "Try again with something else" is a bit wild :) Let's remove it.

Reply:
done :)